lib/device/library/linux.rb

```diff
@@ -17,5 +17,56 @@ def get_device_root_dir(volume_name)
       end
       nil
     end
+
+    # 端末のデバイスファイル名と uhelper オプションの値を取得
+    def device_mount_info(volume_name)
+      device_root = get_device_root_dir(volume_name)
+      raise Device::CantEject, "端末が接続されていません" if device_root.nil?
+
+      pattern = %r!^(/dev/[^ ]+) .* #{device_root} .*\Wuhelper=(\w+)!
+      open("|mount") do |f|
+        while line = f.gets
```

Comment:
__[rubocop]__
* Lint/AssignmentInCondition: Assignment in condition - you probably meant to use `==`. [:ref](https://github.com/bbatsov/ruby-style-guide#safe-assignment-in-condition)

<!-- Comment by [SideCI](https://sideci.com) -->